### DISCUSSION TITLE
Could not load events A document with id `signal-132` was not found.

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [datalust/seq-tickets#1516] [Help]
Greetings,

I just encountered a situation where my Seq instance stopped responding and instead displayed an orange bar with the following message:

```
Could not load events
A document with id `signal-132` was not found.
```

After a while the message went away but now it appears all my API keys and signals ~~, and alerts~~ have been erased. :(

Any guidance/suggestions here would be greatly appreciated.

---

Edit: this is on `Seq 2021.4.7192`

Editx2: Thought alerts were wiped but they are intact, along with the apps they are assigned to. 😌  All events are still in place.  So that means somehow my signals and API keys got erased and/or cannot be located for some reason.

**Comment by liammclennan** [comment]
@Mike-E-angelo please send a diagnostic report to `support@datalust.co`. 

How is your infrastructure configured? In particular, what sort of storage are you using? The issue you have described is known to occur with network file systems.

**Reply by Mike-E-angelo** [reply]
Thank you for your quick reply @liammclennan.  Please note that I am using a Docker image with premium file storage attached in Azure.

I have sent the diagnostics to the requested address.  Let me know if any further information is required to diagnose and I will do my best to assist.
